### ISSUE TITLE
[cssom-view] Default values of ScrollIntoViewOptions contradicts in spec

### ISSUE THREAD

**Issue by syj10** [w3c/csswg-drafts#1353]
The default values of ScrollIntoViewOptions::block and inline are "center" [here](https://www.w3.org/TR/cssom-view-1/#extension-to-the-element-interface) , but are "start" and "nearest" in the 4th step [here](https://www.w3.org/TR/cssom-view-1/#dom-element-scrollintoview).

What would the default values be? Feels like the latter("nearest" and "start") makes more sense?

**Comment by zcorpan** [comment]
The intent is to provide an easy way to do center-center, since that is a common case, but legacy requires nearest-start for no-argument or `true` calls.

i.e.
```
elm.scrollIntoView({}); // center-center
elm.scrollIntoView(); // nearest-start
elm.scrollIntoView(true); // nearest-start
elm.scrollIntoView(false); // nearest-end
```

I would also recommend reading the editor's draft: https://drafts.csswg.org/cssom-view/

**Comment by zcorpan** [comment]
This is the same issue as #1720 and was fixed by #1505.
